ODIN_II/regression_test/benchmark/task/mixing_optimization/cmd_line_options/task.conf

```diff
@@ -0,0 +1,18 @@
+##############################################
```

Comment:
I would change `cmd_line_options` to multipliers_fixed for fixed size architecture, multipliers only.

Reply:
@jeanlego What do you mean fixed size architecture? all of them have ```auto layout```

Reply:
fixed!